mw_api_rate_limit_test.go

```diff
@@ -89,6 +94,128 @@ func TestRLOpen(t *testing.T) {
 	DRLManager.RequestTokenValue = 0
 }
 
+func requestThrottlingTest(t *testing.T) {
+	defer resetTestConfig()
+
+	ts := newTykTestServer()
+	defer ts.Close()
+
+	testLevel := os.Args[0]
+	tn := strings.Split(t.Name(), "/")[1]
+
+	globalCfg := config.Global()
+
+	switch tn {
```

Comment:
Do not rely on the test name. Pass configuration as arguments (see below comments for recommended approach)